mubench.reviewsite/src/Models/MisuseTag.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace MuBench\ReviewSite\Models;
+
+
+use Illuminate\Database\Eloquent\Model;
+
+class MisuseTag extends Model
```

Comment:
This should also be replaced by a many-to-many relation. See `MetadataType` above.